pkg/notifiers/sender.go

```diff
@@ -87,6 +104,96 @@ func TestPagerduty(key, msg string) error {
 	return nil
 }
 
+func TestWechat(secret, agent, corp, receiverType, receiver, msg string) error {
+	req, err := http.NewRequest(http.MethodGet, "https://qyapi.weixin.qq.com/cgi-bin/gettoken", nil)
+	if err != nil {
+		return err
+	}
+
+	q := req.URL.Query()
+	q.Add("corpid", corp)
+	q.Add("corpsecret", secret)
+	req.URL.RawQuery = q.Encode()
+
+	client := &http.Client{
+		Timeout: 15 * time.Second,
+	}
+	var wechatToken wechatToken
+	resp, err := client.Do(req)
+
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+
+	requestBytes, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return err
+	}
+
+	if err := json.Unmarshal(requestBytes, &wechatToken); err != nil {
+		return err
+	}
+
+	if wechatToken.AccessToken == "" {
+		return fmt.Errorf("Invalid APISecret for CorpID. %s", corp)
+	}
+
+	if msg == "" {
+		msg = "Wechat setting validated"
+	}
```

Comment:
Can we move them to the beginning of the the TestWechat func?

if msg == "" {
    msg = "Wechat setting validated"
}